app/controllers/discover.js

```diff
@@ -172,22 +159,23 @@ export default Ember.Controller.extend(Analytics, {
                     hyperLinks: [// Links that are hyperlinks from hit._source.lists.links
                         {
                             type: 'share',
-                            url: config.SHARE.baseUrl + 'curate/preprint/' + hit._id
+                            url: config.SHARE.baseUrl + 'preprint/' + hit._id
                         }
                     ],
                     infoLinks: [] // Links that are not hyperlinks  hit._source.lists.links
                 });
 
-                hit._source.lists.links.forEach(function(linkItem) {
-                    if (isHyperLink(linkItem.url)) {
-                        result.hyperLinks.push(linkItem);
+                hit._source.identifiers.forEach(function(identifier) {
+                    if (identifier.startsWith('http://')) {
+                        result.hyperLinks.push({url: identifier});
                     } else {
-                        result.infoLinks.push(linkItem);
+                        let [type, uri, ..._] = identifier.split('://'); // jshint ignore:line
+                        result.infoLinks.push({type, uri});
                     }
                 });
 
-                result.contributors = result.lists.contributors.map(contributor => ({
-                    users: Object.keys(contributor).reduce((acc, key) => Ember.merge(acc, {[key.camelize()]: contributor[key]}), {})
+                result.contributors = result.lists.contributors.sort((a, b) => (b.order_cited || -1) - (a.order_cited || -1)).map(contributor => ({
+                    users: Object.keys(contributor).reduce((acc, key) => Ember.merge(acc, {[key.camelize()]: contributor[key]}), {bibliographic: contributor.relation !== 'contributor'})
```

Comment:
Looks like a bad parenthesis here. Not sure why the comma and {} was there originally. it should all be one object to merge with acc.

e.g.
```js
                result.contributors = result
                    .lists
                    .contributors
                    .sort((a, b) => (b.order_cited || -1) - (a.order_cited || -1))
                    .map(contributor => ({
                        users: Object
                            .keys(contributor)
                            .reduce((acc, key) => Ember
                                .merge(acc, {
                                    [key.camelize()]: contributor[key],
                                    bibliographic: contributor.relation !== 'contributor'
                                })
                            )
                    }));
```